### ISSUE TITLE
About the training scheme on ShapeNet cars

### ISSUE THREAD

**Issue by kiddoray** [junshengzhou/CAP-UDF#8]
Hi,
would you mind enlightening me that during the experiment on ShapeNet cars, do you train and evaluate on all 3094 cars, or just select a certain number of cars?
It would be of great help to me.
Thanks.

**Comment by junshengzhou** [comment]
Hi,
Thanks for your interest! Since we learn to reconstruct surfaces from a **raw point cloud** without requiring training sets or ground truths. The results under ShapeNetCars dataset are conducted only on the raw point clouds of the test set of  ShapeNetCars with the same split as GIFS, the training set of ShapeNetCars are not used.

Best,
